src/rgw/rgw_kafka.cc

```diff
@@ -449,6 +449,7 @@ class Manager {
     } else {
         ldout(conn->cct, 20) << "Kafka publish (no callback): OK" << dendl;
     }
+    // coverity[RESOURCE_LEAK:FALSE]
```

Comment:
there is no leak. but we should probably use a `unique_ptr` for the tag. and release if `rd_kafka_produce()` was ok